glean/metrics.yaml

```diff
@@ -1657,3 +1657,222 @@ sample:
         description: |
           The line number where the error has been triggered
         type: quantity
+
+  speed_test_started:
```

Comment:
What is this metric trying to answer? Specifically in relation to "speed_test_completed". 

`type:Event`'s are annoying to analyze, and the relation of multiple event's is worse ... or so i have been told by people i'm annoying about relating events for me. 

What do you think about: 
-  have a single metric `speed_test`. 
- Have it a labeled counter: https://mozilla.github.io/glean/book/reference/metrics/labeled_counters.html
- Each state "good", "bad", "failed" , "canceled" ... etc  is a label
We add +1 on the counter of the outcome. 

That way we can calc the sum and and the distribution of each "bucket" including cancelations and failures

Reply:
I'm going to drop the speed_test_started metric. It's not called for by the PRD, and if we end up wanting additional speed test outcomes (that weren't requested in the PRD), we can add them to the speed_test_completed test (as you point out).